src/Durable/Commands/InvokeActivityFunctionCommand.cs

```diff
@@ -1,55 +1,56 @@
-//
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-//
-
-#pragma warning disable 1591 // Missing XML comment for publicly visible type or member 'member'
-
-namespace Microsoft.Azure.Functions.PowerShellWorker.Durable
-{
-    using System.Collections;
-    using System.Management.Automation;
-
-    /// <summary>
-    /// Invoke an activity function.
-    /// </summary>
-    [Cmdlet("Invoke", "ActivityFunction")]
-    public class InvokeActivityFunctionCommand : PSCmdlet
-    {
-        /// <summary>
-        /// Gets and sets the activity function name.
-        /// </summary>
-        [Parameter(Mandatory = true)]
-        public string FunctionName { get; set; }
-
-        /// <summary>
-        /// Gets and sets the input for an activity function.
-        /// </summary>
-        [Parameter]
-        [ValidateNotNull]
-        public object Input { get; set; }
-
-        [Parameter]
-        public SwitchParameter NoWait { get; set; }
-
-        private readonly DurableTaskHandler _durableTaskHandler = new DurableTaskHandler();
-
-        protected override void EndProcessing()
-        {
-            var privateData = (Hashtable)MyInvocation.MyCommand.Module.PrivateData;
-            var context = (OrchestrationContext)privateData[SetFunctionInvocationContextCommand.ContextKey];
-            var loadedFunctions = FunctionLoader.GetLoadedFunctions();
-
-            var task = new ActivityInvocationTask(FunctionName, Input);
-            ActivityInvocationTask.ValidateTask(task, loadedFunctions);
-
-            _durableTaskHandler.StopAndInitiateDurableTaskOrReplay(
-                task, context, NoWait.IsPresent, WriteObject, failureReason => DurableActivityErrorHandler.Handle(this, failureReason));
-        }
-
-        protected override void StopProcessing()
-        {
-            _durableTaskHandler.Stop();
-        }
-    }
-}
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+#pragma warning disable 1591 // Missing XML comment for publicly visible type or member 'member'
+
+namespace Microsoft.Azure.Functions.PowerShellWorker.Durable.Commands
+{
+    using System.Collections;
+    using System.Management.Automation;
+    using Microsoft.Azure.Functions.PowerShellWorker.Durable.Tasks;
+
+    /// <summary>
+    /// Invoke an activity function.
+    /// </summary>
+    [Cmdlet("Invoke", "ActivityFunction")]
+    public class InvokeActivityFunctionCommand : PSCmdlet
+    {
+        /// <summary>
+        /// Gets and sets the activity function name.
+        /// </summary>
+        [Parameter(Mandatory = true)]
+        public string FunctionName { get; set; }
```

Comment:
Should we validate `FunctionName` for null or empty?

Reply:
I believe `Mandatory = true` is already enforcing the check. Furthermore, we later verify that this name actually matches an existing function name.

Reply:
Please also note that this PR shows that the entire file changed just because of line ending unification. The actual meaningful changes are only around namespaces and file locations.